File: VolleyballManagementAppBackend/VolleyballManagementAppBackend/Entities/Team.cs
namespace VolleyballManagementAppBackend.Entities
{
  public class Team
  {
    public Team() { }
    public Team(int id) {
        Id = id;
    }
    public Team(int id, string name, string picture, string description) : this(id)
    {
        Name = name;
        Picture = picture;
        Description = description;
    }

    public int Id { get; set; }
    public string Name { get; set; } = null!;
    public string Picture { get; set; } = null!;
    public string Description { get; set; } = null!;
  }
}

File: VolleyballManagementAppFrontend/src/app/models/entities/team.ts
import { Match } from "./match";
import { Tournament } from "./tournament";
import { User } from "./user";
import { Location } from './location';
import { Training } from "./training";

export interface Team {
  id: string;
  name: string;
  pictureLink: string;
  description: string;
  owner: User;
  location: Location;
  players: User[];
  coaches: User[];
  matches: Match[];
  trainings: Training[];
  tournaments: Tournament[];
  userHasAsFavourite: User[];
}